README.md

```diff
@@ -22,35 +22,49 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 # Contents
 
 - [What is Docker?](#what-is-docker)
-- [Where to start?](#where-to-start)
-- [Where to start? (Windows)](#where-to-start-windows)
-- [Tools](#tools)
-	- [Continuous Integration / Continuous Delivery](#continuous-integration--continuous-delivery)
-		- [CI Services](#ci-services)
-	- [Deployment and Infrastructure](#deployment-and-infrastructure)
-	- [Developer Tools](#developer-tools)
-	- [Development Environments](#development-environments)
-	- [Dockerfile](#dockerfile)
-	- [Garbage Collection](#garbage-collection)
-	- [Hosting Images (registries)](#hosting-images-registries)
-	- [Image Builder](#image-builder)
-	- [Linter / Validator](#linter--validator)
-	- [Local Container Manager](#local-container-manager)
-	- [Monitoring & Logging](#monitoring--logging)
-		- [Monitoring & Logging Services](#monitoring--logging-services)
-	- [Networking](#networking)
-	- [PaaS](#paas)
-	- [Remote Container Manager / Orchestration](#remote-container-manager--orchestration)
-	- [Reverse Proxy](#reverse-proxy)
-	- [Security](#security)
-	- [Serverless](#serverless)
-	- [Service Discovery](#service-discovery)
-	- [CaaS - Services for running containers](#services-for-running-containers)
-	- [Terminal User Interface](#terminal-user-interface)
-	- [Testing](#testing)
-	- [Utilities](#utilities)
-	- [Volume management and plugins](#volume-management-and-plugins)
-	- [Web Interface](#web-interface)
+	- [Where to start?](#where-to-start)
+	- [Where to start? (Windows)](#where-to-start-windows)
+- [Container Operations](#)
+	- [Container Composition](#)
+	- [Terminal UI](#) to help manage docker container from a terminal
+	- [Docker Web UI](#) to help manage docker container from a browser
+	- [Deployment and Infrastructure](#)
+	- [Orchestration](#)
+	 <-- might cause an overlap with PaaS, how to differentiate the 2?
+	- [CaaS](#)
+		 (do we have any project that is not a paying service there? )
+	- [Monitoring](#)
+	- [Networking](#)
+	- [Reverse Proxy](#)
+	- [Security](#)
+	- [Service Discovery](#)
+	- [Volumes Management / Data](#)
+- [Docker Images](#)
+    - [Building](#)
+    	- [Building Blocks](#)
+		 System tools for lack of a better name: project to help containerize an app --> Inside Container Tools ? please help with the naming because System tools means nothing to me.
+	- [Dockerfile](#)
+		 example to help build lean and clean docker images
+	- [Linting](#)
+	- [Registries](#)
+- [Development with Docker](#)
+	- [API Clients](#)
+	- [CI/CD](#)
+	- [Development Environments](#)
+	- [Garbage Collection](#) to keep your dev env clean
+	- [Docker Event Hooks / Sidecars](#)
+	- [Serverless](#)
+	- [Testing](#)
+	- [Workstation Installation](#) <-- For non production usage e.g. macosx
```

Comment:
I think the icons/tags might work better here, Some tools work on more than 1 OS.